cmake_format/commands.py

```diff
@@ -511,11 +525,26 @@ def get_fn_spec():
           "RUN_OUTPUT_VARIABLE": ZERO_OR_MORE
       })
 
+  fn_spec.add(
```

Comment:
Hm... I wonder is it best to include functions that aren't builtin but provided by standard findscripts or listfiles? Lemme think about that...